frontend/src/components/forum/ForumPost.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { ForumPostProps } from "../../types/forum-post";
+import { IconHeart, IconMessageDots } from "@tabler/icons-react";
+import { cx } from "class-variance-authority";
+
+export const ForumPost = ({ title, content, upvote, username, createdAt }: ForumPostProps) => {
```

Comment:
```suggestion
export const ForumPost = ({ post }: {post: ForumPost}) => {
```

and then we can use it as `post.title` etc... without that many props

Reply:
Makes sense.